### ISSUE TITLE
Numix Light - Nautilus tabs are black (using GTK3 version)

### ISSUE THREAD

**Issue by chris-asl** [numixproject/numix-gtk-theme#211]
Hello, 

I've just purchased the Numix Light theme for my Ubuntu 14.04 system, using Unity DE. I've extracted both folders and tested them on their appearance.  
**I'm using the numix-circle-light icon theme**.   

**GTK-3 folder**, seems to be working just fine except for the tabs at Nautilus for a non-root user are black, as shown below:  
![nautilus-black](https://cloud.githubusercontent.com/assets/2846837/7333780/a71faf38-eb84-11e4-86ed-2528bad054ee.png)

When starting Nautilus with root privileges (`sudo nautilus`), the tabs look just fine.  

**GTK-3.4 folder**, the tabs at Nautilus work fine for a non-root user, **but**, there are highlights at the launcher icons:  
![launcher-icons](https://cloud.githubusercontent.com/assets/2846837/7333815/4a7f3536-eb85-11e4-9868-e262a37d5a76.png)

Also, the window switcher highlight effect looks different:  
![window-switcher](https://cloud.githubusercontent.com/assets/2846837/7333825/83bbd110-eb85-11e4-8088-718edc53b1bb.png)

**Comment by Corsseir** [comment]
You didnt install theme properly. Please use launchpad ppa for it:
sudo apt-add-repository -y ppa:numix/ppa
sudo apt-get install numix-gtk-theme numix-icon-theme-circle

**Comment by chris-asl** [comment]
Thank you for the reply.
I've added the ppa you've provided, tried installing these two packages, and they were already the latest version.
So, I purged them (the `numix-icon-theme` package, as well) and then re-installed them.  

_The problem with the window switcher was resolved_, **but** the problem with the tabs of Nautilus being black persists (tabs look fine when Nautilus has root privileges, but look black when non-root privileges).  

Also, are you sure, this installs Numix Light? Because, this is the theme I'm interested in installing.
